style.css

```diff
@@ -1 +1,146 @@
-/*  Add your styling here */
+.header{
+    width: 100%;
+    top: 0;
+    left: 0;
+    right: 0;
```

Comment:
These `0` values are not necessary as they are the default, they don't do any harm, but will make me wonder why you've used them if I look at them in the future...